CMakeLists.txt

```diff
@@ -75,7 +75,6 @@ include(iree_py_extension)
 include(iree_py_library)
 include(iree_py_test)
 include(iree_lit_test)
-include(iree_alwayslink)
```

Comment:
:tada: !!!!!

Reply:
Be aware that this functions/module was only required for *external* deps. Internal targets are still linked as whole archive.

Reply:
Ah. Still nice though :-D